routers/init.go

```diff
@@ -188,5 +189,9 @@ func NormalRoutes() *web.Route {
 	r.Mount("/", web_routers.Routes(sessioner))
 	r.Mount("/api/v1", apiv1.Routes(sessioner))
 	r.Mount("/api/internal", private.Routes())
+	r.Get("/api/_healthz", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
```

Comment:
Shouldn't it do actual health check, let's say at least ping database?

Reply:
the whole thing is for container orchestration to check the app is live.
it can be in different level. this is a simple `200` code response, the very basic level says that: hey, I'm ready and I'm not down. in most case this is enough.

we can also add `redis` ping and `database` ping check, but this will have a little overhead.

Reply:
currently, I'm using `/assets/img/logo.svg?from-health-check` as the check path.
Because 1. it does not need api authentication. simple to deploy  2. logo.svg response in text and has small body.

But, it looks very confusing. so I think we need a simple healthz endpoint like sentry, grafana, k8s etc...

Reply:
traefik has `/ping`, and it just returns 200 code and an ok just like this.
do you mind if I change the path to `/api/_ping`  or `/api/ping` ?

Reply:
I'm for `/ping` as it only return ok ...

Reply:
ok, I've changed it to `/api/ping`, we can not use `/ping` because this need to avoid username confliction.

Reply:
haha yes i just did now write the prefix as it was clear to me :D